.jenkins/staticanalysis.groovy

```diff
@@ -65,7 +65,7 @@ def runCompileCommand(platform, project, jobName, boolean debug=false)
 
             popd
 
-            doxygen docs/Doxyfile
+            doxygen docs/doxygen/Doxyfile
```

Comment:
Do we have any doxygen comments to process?

Reply:
Not at the moment, at least via Sphinx. The static analysis CI job still runs the doxygen command however, and the Doxyfile was moved.